frontend/public/actions/ui.ts

```diff
@@ -18,6 +19,7 @@ import { OverviewSpecialGroup } from '../components/overview/constants';
 import { setClusterID, setCreateProjectMessage, setUser, setConsoleLinks } from './common';
 
 export enum ActionType {
+  SetColumnManagementFilter = 'setColumnManagementFilter',
```

Comment:
Is this really a "filter"? Maybe `setTableColumns`?

Reply:
@spadgett i was thinking its similar to our existing rowFilters, instead we are filtering the view from the column perspective, but I can change to setTableColumns.